### PR TITLE
Update OCP logo

### PR DESCRIPTION
**Pull request by aguilerapy**
#51

### CODE REVIEW

**Comment by romifz** [comment]
@aguilerapy please add a screenshot of the main page with the new logo here.

**Comment by aguilerapy** [comment]
Screenshot_2019-11-19 OCDS Web Tools - Home

@romifz this is the screenshot of main page with the new logo

**Comment by romifz** [comment]
@jpmckinney what do you think of keeping the new logo on the top of the page?

**Comment by jpmckinney** [comment]
I think it's a bit heavy. I'd rather put it in the footer.

**Comment by aguilerapy** [comment]
Screenshot_2019-11-26 OCDS Toucan

Screenshot_2019-11-26 OCDS Web Tools - Home

What do you think about this? I can modify the header or add more stuff to the footer if something is wrong @romifz @jpmckinney

**Comment by jpmckinney** [comment]
That works for me!

File: default/static/css/custom.css [hunk 1]
```diff
@@ -65,4 +57,27 @@
...
     padding-top: 3px;
-}
\ No newline at end of file
+}
+.footer.logo {
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 100px;
+    background-color: #222;
```
Comment (aguilerapy) [inline default/static/css/custom.css]: @jpmckinney The CSS changes for the footer color here.

Comment (jpmckinney) [reply default/static/css/custom.css]: Aha, I think it was just a browser cache issue.

**Pull request merged**